### PR TITLE
Enable packit RPM builds on pull requests

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+actions:
+  get-current-version:
+    - "python3 ./setup.py --version"
```
Comment (jkonecny12) [inline .packit.yaml:3]: You should be able to replace this by https://packit.dev/docs/configuration/#upstream_tag_template.

```
upstream_tag_template: blivet-{version}-1
```

**Review by jkonecny12 — COMMENTED** [review]
Just a small improvement.

File: .packit.yaml (hunk 1 above)
Comment (vojtechtrefny) [reply .packit.yaml:3]: That's probably not going to work for us with the correct version only on the `-release` branch.

**Review by dwlehman — APPROVED** [review]
Nice! Looks good to me.

File: .packit.yaml (hunk 1 above)
Comment (jkonecny12) [reply .packit.yaml:3]: OK ;)

**Pull request merged**